person_class.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
👏 Great job in implementing `attr_accessor` for `name` and `age`! 👏 Now, getters and setters for `@name` and `@age` are in place. However, considering that `id` is generated and may not need a setter, let's optimize it by removing it from `attr_accessor` and using `attr_reader`:

- [ ] **Optimizing `id` Attribute:** Since the `id` is generated, it may not require a setter method. I recommend removing `id` from `attr_accessor` and using `attr_reader` to make it a getter-only method.

By making this adjustment, your code will be more concise and align with best practices for attribute handling. Keep up the excellent work! 🚀🔍